packages/test-complexdialogs/models/.metadata

```diff
@@ -0,0 +1 @@
+119d31c870061d98828df425d0e139a79dcd5478e2fef3f18971222931e49e46
```

Comment:
what is this?

Reply:
This is a hash computed to know if the model is up to date